MonoGame.Framework/Graphics/Effect/SpriteEffect.cs

```diff
@@ -1,24 +1,13 @@
-#region File Description
-//-----------------------------------------------------------------------------
-// SpriteEffect.cs
-//
-// Microsoft XNA Community Game Platform
-// Copyright (C) Microsoft Corporation. All rights reserved.
-//-----------------------------------------------------------------------------
```

Comment:
@KonajuGames - Not sure how we should deal with this copyright?  Don't think we can/should remove it.